<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vsourceDialog</class>
 <widget class="QDialog" name="vsourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>253</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Voltage Source</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>12</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>               Mode :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>10</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFreq">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>88</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>   Frequency[Hz] :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>43</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="amp">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>57</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAmp">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>57</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude[V] :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="freq">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>88</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phi">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>119</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhi">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>119</y>
     <width>83</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>              Φ[°] :</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>150</y>
     <width>152</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>176</y>
     <width>152</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vsourceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vsourceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
